public/index.html

```diff
@@ -17,7 +17,7 @@
         <div class="row">
             <div class="col-sm-12">
                 <a ng-href="#/" class="svg logo" ng-click="searchCtrl.clear()">
-                    <object type="image/svg+xml" data="logo.svg">Your browser does not support SVG</object>
+                    <img src="logo.svg"></img>
                 </a>
             </div>
         </div>
```

Comment:
Okay since we're only using a single (known) browser, but worth keeping in mind: http://www.sitepoint.com/add-svg-to-web-page/